src/Grok.Net/Grok.cs

```diff
@@ -38,21 +39,34 @@ public Grok(string grokPattern)
         /// </summary>
         /// <param name="grokPattern">The Grok pattern to use.</param>
         /// <param name="customPatterns">A stream containing custom patterns.</param>
-        public Grok(string grokPattern, Stream customPatterns)
-            : this(grokPattern)
+        public Grok(string grokPattern, Stream customPatterns) : this(grokPattern)
         {
-            LoadCustomPatterns(customPatterns);
+            LoadCustomPatternsFromStream(customPatterns);
         }
 
         /// <summary>
         ///     Initializes a new instance of the <see cref="Grok"/> class with the specified Grok pattern and custom patterns.
         /// </summary>
         /// <param name="grokPattern">The Grok pattern to use.</param>
         /// <param name="customPatterns">Custom patterns to add.</param>
-        public Grok(string grokPattern, IDictionary<string, string> customPatterns)
-            : this(grokPattern)
+        public Grok(string grokPattern, IDictionary<string, string> customPatterns) : this(grokPattern)
         {
-            AddCustomPatterns(customPatterns);
+            LoadCustomPatternsFromDictionary(customPatterns);
+        }
+
+        /// <summary>
+        /// Initialized a new instance of the <see cref="Grok"/> class with specified Grok pattern,
```

Comment:
Look at the example below. There are some spaces at the beginning. Please do the same.

Reply:
Will do.